### PR TITLE
Feat/fwlt 758 verify email on signup

### PR DESCRIPTION
**Pull request by milan-bc**
## Description (optional)
Add a concise explanation of the changes.

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

### CODE REVIEW

**Review by schnogz — APPROVED** [review]

File: packages/blockchain-wallet-v4-frontend/src/data/analytics/types.ts [hunk 1]
```diff
@@ -1,13 +1,13 @@
 import * as AT from './actionTypes'
 import { AB_TESTS } from './model'
-import { MempoolFeeType } from 'data/components/types'
+import { MempoolVerifyEMailType } from 'data/components/types'
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/data/analytics/types.ts:3]: VerifyEmail

File: packages/blockchain-wallet-v4-frontend/src/data/analytics/types.ts [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
 import { RemoteDataType } from 'core/types'
 
 // types
 export type ABTestNameType = AB_TESTS
 
 export type ABTestCmdType = {
-  command: MempoolFeeType
+  command: MempoolVerifyEMailType
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/data/analytics/types.ts:10]: VerifyEmail

File: packages/blockchain-wallet-v4-frontend/src/data/auth/sagas.js [hunk 3]
```diff
@@ -127,16 +127,17 @@ export default ({ api, coreSagas }) => {
...
 
-      const showVerifyEmail = yield select(
+      const showVerifyEmailR = yield select(
         selectors.analytics.selectAbTest(AB_TESTS.VERIFY_EMAIL)
       )
+      const showVerifyEmail = showVerifyEmailR.getOrElse({})
 
       if (
-        Remote.success.is(showVerifyEmail) &&
+        showVerifyEmail &&
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/data/auth/sagas.js:136]: Remote.Success.is ?

File: packages/blockchain-wallet-v4-frontend/src/scenes/VerifyEmail/template.tsx [hunk 4]
```diff
@@ -22,8 +22,6 @@ const IconWrapper = styled.div<{ color: keyof DefaultTheme }>`
   margin-right: 1.25rem;
 
   ${media.tablet`
-    height: 2.5rem;
-    width: 2.5rem;
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/scenes/VerifyEmail/template.tsx:26]: No rems does not mean no styling, you can use pxs

**Pull request merged**